tamr_client/_types/backup.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+from tamr_client._types.url import URL
+
+
+@dataclass(frozen=True)
+class Backup:
+    """A Tamr backup
+
+    See https://docs.tamr.com/new/docs/configuration-backup-and-restore
+
+    Args:
+        url
+        resource_id
+        path
+        state
+        errorMessage
+    """
+
+    url: URL
+    resource_id: str
+    path: str
+    state: str
+    errorMessage: str
```

Comment:
camelCase -> snake_case:

```suggestion
    error_message: str
```